lib/shared/repositories/givt_repository.dart

```diff
@@ -107,4 +109,9 @@ class GivtRepositoryImpl with GivtRepository {
   @override
   Future<bool> deleteGivt(List<dynamic> ids) async =>
       apiClient.deleteGivts(body: ids);
+
+  @override
+  Future<void> downloadYearlyOverview(String guid, String year) async {
+    await apiClient.downloadYearlyOverview(guid, year);
```

Comment:
isn't this supposed to return a bool as in the "apiClient.downloadYearlyOverview(guid, year)"?

Reply:
Yeah